trie/zk/tree_path.go

```diff
@@ -0,0 +1,72 @@
+package zk
+
+import (
+	zkt "github.com/kroma-network/zktrie/types"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// TreePath 0 left, 1 right
+type TreePath []byte
+
+const (
+	left  = byte(0)
+	right = byte(1)
+)
+
+func NewTreePathFromHash(hash common.Hash) TreePath {
+	return NewTreePathFromBytesAndMaxLevel(zkt.ReverseByteOrder(hash[:]), len(hash)*8)
+}
+
+func NewTreePathFromZkHash(hash zkt.Hash) TreePath {
+	return NewTreePathFromBytesAndMaxLevel(hash[:], len(hash)*8)
+}
+
+func NewTreePathFromBytes(bytes []byte) TreePath {
+	return NewTreePathFromBytesAndMaxLevel(bytes, len(bytes)*8)
+}
+
+func NewTreePathFromBytesAndMaxLevel(bytes []byte, maxLevel int) TreePath {
+	path := make(TreePath, maxLevel)
+	for n := 0; n < maxLevel; n++ {
+		if zkt.TestBit(bytes[:], uint(n)) {
+			path[n] = right
+		} else {
+			path[n] = left
+		}
+	}
+	return path
+}
+
+func NewTreePath(path []byte) TreePath { return path }
+
+func (p TreePath) Get(depth int) byte { return p[depth] }
+func (p TreePath) GetOther(depth int) byte {
+	if p[depth] == left {
+		return right
+	}
+	return left
```

Comment:
we can simply use bit operator for this. 
```suggestion
	return p[depth] ^ byte(1)
```